app/assets/javascripts/quby/answers/rest.js

```diff
@@ -318,23 +323,24 @@ function handleAjaxFormRequests() {
 
 var leave_page_text;
 $(function() {
-        $('input').placeholder();
-
-        setupLeavePageNag();
-
-        $(".deselectable").deselectable();
+    $('input').placeholder();
 
-        $('input[type="radio"]:not(.subinput), input[type="checkbox"]:not(.subinput)').on("click", radioCheckboxEvents );
+    setupLeavePageNag();
 
-        processExtraData();
+    $(".deselectable").deselectable();
 
-        if (isBulk) {
-            prepareBulk();
-        } else {
+    $('input[type="radio"]:not(.subinput), input[type="checkbox"]:not(.subinput)').on("click", radioCheckboxEvents );
 
-            handleAjaxFormRequests();
+    processExtraData();
 
-            preparePaged();
-        }
+    if (displayMode == 'bulk') {
```

Comment:
Als je nou 1 functie prepareJquerySpaghetti maakt die op basis van de displayMode de juiste functie aanroept? Voor bulk versie mag je ook we handleAjaxFormRequests aanroepen, gezien die toch geen ajax gebruikt (en als hij dat deed wil je waarschijnlijk prepareBulk aanroepen).